builder/optimizations.cc

```diff
@@ -359,6 +359,8 @@ class copy_aliaser : public stmt_mutator {
       alias.assume_in_bounds = in_bounds;
     }
 
+    // contradictions when we do so. This tracks what we want the strides to be.
```

Comment:
It seems like a part of this comment is missing.

Reply:
Huh, that file wasn't saved. Fixed.